File: mRemoteNG/Connection/Connection.Protocol.SSH1.vb
Namespace Connection
    Namespace Protocol
        Public Class SSH1
            Inherits Connection.Protocol.PuttyBase

            Public Sub New()
                Me.PuttyProtocol = Putty_Protocol.ssh
                Me.PuttySSHVersion = Putty_SSHVersion.ssh1
            End Sub

            Public Enum Defaults
                Port = 22
            End Enum
        End Class
    End Namespace
End Namespace

File: mRemoteV1/Connection/Protocol/SSH/Connection.Protocol.SSH1.cs
namespace mRemoteNG.Connection.Protocol.SSH
{
    public class ProtocolSSH1 : PuttyBase
    {
        public ProtocolSSH1()
        {
            PuttyProtocol = Putty_Protocol.ssh;
            PuttySSHVersion = Putty_SSHVersion.ssh1;
        }

        public enum Defaults
        {
            Port = 22
        }
    }
}